<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>370</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NS Team</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/assets/images/micro.png</normaloff>:/assets/images/micro.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="layoutCustomer">
       <item>
        <layout class="QVBoxLayout" name="layoutCustomerAvatar">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutCustomerAvatar">
           <item>
            <spacer name="spacerLeftCustomer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelCustomerAvatar">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/assets/images/avatar.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerRightCustomer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelCustomerText">
         <property name="text">
          <string>Other(s)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutCustomerLangBox">
         <item>
          <spacer name="spacerLeftCustomerLang">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboCustomerLang">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Japanese (日本)</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/assets/images/jp.png</normaloff>:/assets/images/jp.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>English (English)</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/assets/images/uk.png</normaloff>:/assets/images/uk.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vietnamese (Việt Nam)</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/assets/images/vn.png</normaloff>:/assets/images/vn.png</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="spacerRightCustomerLang">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutMic">
       <item>
        <layout class="QVBoxLayout" name="layoutMicInner">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutMic">
           <item>
            <spacer name="spacerLeftMic">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelMic">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/assets/images/micro.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerRightMic">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutToggleButton">
         <item>
          <spacer name="spacerLeftToggle">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonMicToggle">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Live Translate: OFF</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerRightToggle">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutMe">
       <item>
        <layout class="QVBoxLayout" name="layoutMeAvatar">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutMeAvatar">
           <item>
            <spacer name="spacerLeftMe">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelMeAvatar">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/assets/images/avatar.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerRightMe">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelMeText">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>User</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutMyLangBox">
         <item>
          <spacer name="spacerLeftMyLang">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboMyLang">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Vietnamese (Việt Nam)</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/assets/images/vn.png</normaloff>:/assets/images/vn.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>English (English)</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/assets/images/uk.png</normaloff>:/assets/images/uk.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Japanese (日本)</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/assets/images/jp.png</normaloff>:/assets/images/jp.png</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="spacerRightMyLang">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutNewAndSetting">
     <item>
      <widget class="QPushButton" name="buttonNewConversation">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>NEW Conversation</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerBetweenNewAndSetting">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSettings">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Settings</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/assets/images/settings.png</normaloff>:/assets/images/settings.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QTextEdit" name="textChatBox">
       <property name="placeholderText">
        <string>Meeting chats...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
